src/network/addrthread.py

```diff
@@ -5,10 +5,10 @@
 
 # magic imports!
 import state
+import network
```

Comment:
why not `from network import connectionpool`?

Reply:
because it will create a new reference/new copy of connectionpool obj which would be different from network one.